generated/story-ids.json

```diff
@@ -0,0 +1,91 @@
+[
```

Comment:
Are we sure we want to commit to the repo a statically generated file?

Reply:
yeah, I need to gitignore this! (This is just a demo PR, so it's messy)